static/templates/management.html

```diff
@@ -69,6 +76,38 @@
     </form>
 
 <script>
+        $(document).ready(function(){
+                $('#grid').css('visibility', 'hidden')      
+	})
+	function showSsids() {
+	    $('#login').css('visibility', 'hidden');      
+	    $('#grid').css('visibility', 'visible');     
+	}
+	function authenticate(){
+            var pw = $('#password').val();
+	    if ( pw.length < 8) {
+		alert("Password must be at least 8 characters long");
+	    } else {
+	        $.ajax({
+	          type: "POST",
+	          url: "/hashit",
+	          data: {Hash: pw}
```

Comment:
I've verified using tcpdump that password can be captured easily as it travels in plain through network:

POST /hashit HTTP/1.1
Host: elsa.local:8080
User-Agent: Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:53.0) Gecko/20100101 Firefox/53.0
Accept: */*
Accept-Language: en-US,en;q=0.5
Accept-Encoding: gzip, deflate
Content-Type: application/x-www-form-urlencoded; charset=UTF-8
X-Requested-With: XMLHttpRequest
Referer: http://elsa.local:8080/
Content-Length: 13
Connection: keep-alive

Hash=federico

I'm not a security expert, but I think it is not secure generating the hash in javascript side and, besides, that forces to have a static salt there. So, probably we could consider using ssl. 
We can think about this. Maybe Let's encrypt can be a good solution (though requires internet connection to generate the certificate)